src/rust/ide/lib/enso-protocol/src/project_manager.rs

```diff
@@ -106,6 +110,18 @@ pub struct ProjectMetadata {
     pub last_opened : Option<UTCDateTime>
 }
 
+/// This type specifies what action should be taken if a component required to complete an operation
```

Comment:
What kind of component? What kind of operation?

Reply:
Component = Engine's component
Operation = any operation taking this type as a parameter.

I will add the explanation in text PR